packages/fxa-payments-server/server/lib/server.js

```diff
@@ -27,31 +27,28 @@ module.exports = () => {
   // Each of these config values (e.g., 'servers.content') will be exposed as the given
   // variable to the client/browser (via fxa-content-server/config)
   const CLIENT_CONFIG = {
+    sentryDsn: config.get('sentryDsn'),
     servers: {
       auth: {
-        url: config.get('servers.auth.url'),
+        url: config.get('servers.auth.url')
```

Comment:
Sigh. I used `eslint . --fix` to put the quotes back how they were and in doing so it's changed all this other stuff too. One sec: I'll checkout, re-apply then force push this with just my changes.